src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {
+  ChakraProvider,
+  extendTheme,
+} from '@chakra-ui/react';
+
+import { HomePage } from './Pages';
+
+function App() {
+  return (
+    <ChakraProvider theme={extendTheme({
+      fonts: {
+        body: `'Open Sans', sans-serif`,
+      },
```

Comment:
It would be better if the font stayed consistent with what is already used in the Tab Butler extension itself.

Reply:
Is it Roboto in the extension?

Reply:
@mohitkyadav No for the font we use the system font (Segoe UI for Windows and SF Pro for Mac and a few others). We do that using this font-family: `-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
              Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;`

Here is a guide that explains it better: https://www.digitalocean.com/community/tutorials/css-system-font-stack